packages/coordinator/src/components/CsvDownloaderButton/CsvDownloaderButton.tsx

```diff
@@ -0,0 +1,41 @@
+import CsvDownloader from "react-csv-downloader";
+import { IColumn } from "react-csv-downloader/dist/esm/lib/csv";
+import Button from "../Button";
+
+export interface ZMCsvDownloaderButtonProps {
+  columns?: IColumn[];
+  data: { [key: string]: any }[];
+  fileName: string;
+  separator?: string;
```

Comment:
In my opinion, if no one uses a different value than `,` we should not expose it.

We can always add this when needed